content/body/switch.php

```diff
@@ -2,13 +2,13 @@
   A switch is like a checkbox, in that is designed to be an input control that has a binary value
   (either <strong>checked</strong> and <strong>unchecked</strong> or <strong>on</strong> or <strong>off</strong>,
   depending on the screen reader).
-  Like <a href="tabs.php">tablists.php</a>, switches do not have a native HTML5 tag, so we implement custom code using
+  Like <a href="tabs.php">tablists.php</a>, switches/Toggles do not have a native HTML5 tag, so we implement custom code using
```

Comment:
We need to be consistent with casing:

```suggestion
  Like <a href="tabs.php">tablists.php</a>, switches/toggles do not have a native HTML5 tag, so we implement custom code using
```